package.json

```diff
@@ -4,17 +4,19 @@
   "description": "Twilio Video Room Monitor Javascript Library",
   "main": "dist/node/index.js",
   "types": "dist/node/index.d.ts",
+  "regular": "dist/browser/twilio-video-room-monitor.js",
```

Comment:
I think `browser` would be a more descriptive target name than `regular`.

Reply:
Looks like `browser` is reserved for library targets, and we aren't able to use the `scopeHoist` option with this. Instead of using `browser`, I changed it to `browserTarget` and `minifiedTarget` to be uniform but let me know if you have another idea!